code/script.js

```diff
@@ -42,46 +41,47 @@ const sunsetIcon = createWeatherImg("./assets/sunset-icon.png", "sunset");
 
 
 function createFiveDayForecast(filteredForecast) {
+    weatherContainer.innerHTML = ""
     for (let i = 0; i < 5; i++) {
 
-        chooseWeatherIcon(filteredForecast, i);
+        const daysWeatherIcon = chooseWeatherIcon(filteredForecast[i]);
         const days = new Date(filteredForecast[i].dt_txt).toLocaleDateString(
             "en-US",
             { weekday: "long" }
         );
         const forecastTemp = Math.round(filteredForecast[i].main.temp * 10) / 10;
         const forecastWeather = filteredForecast[i].weather.description;
-        createFiveDaysInnerHTML(days, forecastTemp);
-    }
+        weatherContainer.innerHTML += createFiveDaysInnerHTML(days, forecastTemp, daysWeatherIcon);
+    };
 };
 
 //Creates a weather icon based on the forecasted weather
-function chooseWeatherIcon(filteredForecast, i) {
```

Comment:
this function didn't actually need two arguments, so we've just given it each day's forecast (from 'filteredForcast[i]) instead